features/release.feature

```diff
@@ -367,6 +367,36 @@ MARKDOWN
           ASSET_TARBALL
           """
 
+  Scenario: Delete a release
+    Given the GitHub API server:
+      """
+      get('/repos/mislav/will_paginate/releases') {
+          json [
+            { url: 'https://api.github.com/repos/mislav/will_paginate/releases/123',
+              upload_url: 'https://api.github.com/uploads/assets{?name,label}',
+              tag_name: 'v1.2.0',
+              name: 'will_paginate 1.2.0',
+              draft: true,
+              prerelease: false,
+              assets: [
+                { url: 'https://api.github.com/repos/mislav/will_paginate/assets/9876',
+                  name: 'hello-1.2.0.tar.gz',
+                },
+              ],
+            },
+          ]
+      }
+
+      delete('/repos/mislav/will_paginate/releases/123') {
+        status 204
+      }
+      """
+    When I successfully run `hub release delete v1.2.0`
+    Then the output should contain exactly:
+      """
+      Deleting release related to v1.2.0...\n
```

Comment:
Oh sure, you already added code that verifies that 204 was returned. That part is fine. However, if the `hub release delete` command has no output, and exists with a success status, the test still can't verify that it actually did anything, i.e. that it did issue a DELETE request. That's why I suggested a bit of trickery with the test server.

My suggested code above could work, but it hasn't been tested. If you'd like me to improve the test instead, I can definitely take over once you're done with other things.